corehq/apps/export/tasks.py

```diff
@@ -37,11 +38,13 @@
 
 
 @task(serializer='pickle', queue=EXPORT_DOWNLOAD_QUEUE)
-def populate_export_download_task(export_instances, filters, download_id, filename=None, expiry=10 * 60):
+def populate_export_download_task(domain, export_ids, exports_type, username, filters, download_id,
```

Comment:
I think that this will cause all exports that are currently queued during deploy, but not yet started. What I mean by this is:

1. An export is queued by someone, meaning a message is sent to rabbitmq with arguments `(export_instances, filters, download_id, filename=None, expiry=10 * 60)`
2. We begin a deploy
3. Services restart
4. Export from 1 is read by a new celery worker running this new code. The arguments have changed so the new code cannot run the old export
5. customer gets a sad face boy
6. support gets an angry email we can't reproduce

To work around this, you can:

1. Keep this task the same as the old format
2. Create a new task `new_populate_export_download_task` with this new format
3. deploy that
4. wait a day/deploy cycle so that all old exports are out of the queue
5. Remove `populate_export_download_task`

Preferably there would be some renames better naming that `new_xxx_task` invovled

Reply:
Good point.. considering [how many exports are run](https://app.datadoghq.com/dashboard/bdu-k5b-bz5/celery-overview?screenId=bdu-k5b-bz5&screenName=celery-overview&from_ts=1552296120000&is_auto=false&live=true&page=0&tile_size=m&to_ts=1552299720000&fullscreen_widget=185712938) i thought the probability of a task getting stuck like that would be highly improbable.

Reply:
After seeing that graph, is it fine if i just keep an eye on any errors just after deploy instead? I think the error would make it to sentry so should be easy to track..

Reply:
So it looks like there's at most one in the queue? that seems reasonable.